templates/api.html

```diff
@@ -83,53 +84,60 @@ <h1>Examples</h1>
 <br>
 Search<br>
 <a href="https://api.cdnjs.com/libraries?search=ractive">https://api.cdnjs.com/libraries?search=ractive</a><br>
-<pre><code class="json">{"results":[{"name":"ractive","latest":"https://cdnjs.cloudflare.com/ajax/libs/ractive/0.8.5/ractive.min.js"},{"name":"ractive.js","latest":"https://cdnjs.cloudflare.com/ajax/libs/ractive.js/0.3.7/ractive.min.js"},{"name":"ractive-require","latest":"https://cdnjs.cloudflare.com/ajax/libs/ractive-require/0.6.12/ractive-require.min.js"}],"total":3}</code></pre>
+<pre><code class="json">{"results": [{"name": "ractive","latest": "https://cdnjs.cloudflare.com/ajax/libs/ractive/0.8.5/ractive.min.js","sri": "sha256-HMVN0791sxknofB+JpPiQcIx6kfBAiKruKmZUvD306I="},{"name": "ractive.js","latest": "https://cdnjs.cloudflare.com/ajax/libs/ractive.js/0.3.7/ractive.min.js","sri": "sha256-vawtizVRhXOHO+bDa62d11fEGS7p7pHSFXlezXX4Ca8="},{"name": "ractive-require","latest": "https://cdnjs.cloudflare.com/ajax/libs/ractive-require/0.6.12/ractive-require.min.js","sri": "sha256-d8adML5s1KGmpmJke92gqFaPzHU/StSaA0BJxRzvTIU="}],"total": 3}</code></pre>
 <br>
 Search Human Readable output<br>
-<a href="https://api.cdnjs.com/libraries?search=jquery&output=human">https://api.cdnjs.com/libraries?search=jquery&output=human</a><br>
+<a href="https://api.cdnjs.com/libraries?search=ractive&output=human">https://api.cdnjs.com/libraries?search=ractive&output=human</a><br>
```

Comment:
Can we keep `jquery` here?

Reply:
Would you also like me to change the results to be jQuery instead of reactive?

Reply:
yeah sure, the point here is to keep less diff